extensions/postgres/transactional.go

```diff
@@ -106,6 +106,34 @@ func (d dbTx) UpdateAsyncStateExecution(
 	return nil
 }
 
+const updateAsyncStateWaitUntilToAbortRunningQuery = `UPDATE xdb_sys_async_state_executions set
+version = version + 1,
+wait_until_status = 5
+WHERE process_execution_id=$1 AND wait_until_status=1
+`
+
+const updateAsyncStateExecutionToAbortRunningQuery = `UPDATE xdb_sys_async_state_executions set
+version = version + 1,
+execute_status = 5
+WHERE process_execution_id=$1 AND execute_status=1
+`
+
+func (d dbTx) UpdateAsyncStateExecutionToAbortRunning(
```

Comment:
I see another implementation is [here](https://github.com/xdblab/xdb/blob/e8b9ad0c202d80367a380c0ad218a5ea64defa64/persistence/sql/sql_process_store.go#L451).